pytest.ini

```
@@ -5,3 +5,16 @@ env =
     # pragma: allowlist nextline secret
     D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/fsd_account_store
     GITHUB_SHA=123123
+
+markers =
+    user_config: used to provide users to tests
+
+filterwarnings =
+    error
+
+    # ignore warnings from connexion
+    ignore::DeprecationWarning:connexion\.json_schema
+    ignore::DeprecationWarning:connexion\.validators
```

Depending on which way we go with this, it could just be:

```suggestion
    ignore::DeprecationWarning:connexion
```

---

I re-wrote it to mention each ignored error separately.